src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/CompletionHandler.cs

```diff
@@ -324,7 +324,11 @@ private CompletionContext RewriteContext(CompletionContext context, RazorLanguag
             var invokeKind = (context as VSCompletionContext)?.InvokeKind;
             if (invokeKind.HasValue)
```

Comment:
Even if InvokeKind doesn't have a value shouldn't we potentially be setting invokeKind? I think it might be a better check to do a cross-check of "trigger characters" against Razor's trigger characters because at this point in the method we're effectively only operating on them: https://github.com/dotnet/aspnetcore-tooling/pull/3155/files#diff-052d5834149367e2bcf4d95d0740ddb4b346116cfcd64e568d0b5eca38f7b8aeR22